_posts/2020-12-02-welcome-new-hire-getting-started-in-frontend-at-rimdev-updated.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: "Welcome New Hire: Getting started in Frontend at RIMdev (updated)"
+date: 2020-12-02 08:09:18
+tags:
+- Team
+- RIMdev
+- Frontend
+categories: Team
+twitter_text: "Welcome New Hire: Getting started in #Frontend at RIMdev (update)"
+authors: Kevin Hougasian
+image: https://images.unsplash.com/photo-1524530514346-eab43224489b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2800&q=80
+image_url: https://unsplash.com/photos/AqrD5QQ0ajM
+image_credit: "@ilumire"
+---
+
+<div class="ui info message">
+This is an updated post of the original <a href="/welcome-new-hire-getting-started-in-frontend-at-rimdev/">here</a>.
+</div>
+
+Whether you're new to the frontend at RIMdev or looking for more about our Frontend team, here's a quick overview of our team, tools, and [culture](https://github.com/ritterim/culture)!
+
+As a whole, RIMdev has always strived to use the technology that best suits the task. This post will give you a glimpse into our environment and tooling. Keep in mind, software changes and so do we.
+
+The majority of our frontend team are Mac users, so this post will seem Mac-biased, but if Windows is where you live, that's just fine! We have several Linux enthusiasts across the team as well, however, we don't support it for business use (that includes Linux subsystem for Windows).
```

Comment:
Is it just fine? I mean, there are lines...

Reply:
I feel like Kevin only included that line to taunt me and remind me that I switched sides

Reply:
"just fine" doesn't make it the "right" choice 😜😂

Reply:
haha, sorry @Azbury 🤣